src/main/java/frc/trigon/robot/commands/Commands.java

```diff
@@ -70,4 +84,125 @@ public static Command duplicate(Command command) {
+                ShooterCommands.getReachTargetShootingVelocityFromShootingCalculationsCommand(),
+                SwerveCommands.getClosedLoopFieldRelativeDriveCommand(
+                        () -> CommandConstants.calculateDriveStickAxisValue(OperatorConstants.DRIVER_CONTROLLER.getLeftY()),
+                        () -> CommandConstants.calculateDriveStickAxisValue(OperatorConstants.DRIVER_CONTROLLER.getLeftX()),
+                        () -> SHOOTING_CALCULATIONS.getTargetShootingState().targetRobotAngle()
+                )
+        );
+    }
+
+    public static Command getWarmSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                getUpdateShootingCalculationsCommand(false),
+                PitcherCommands.getReachTargetPitchFromShootingCalculationsCommand(),
+                ShooterCommands.getReachTargetShootingVelocityFromShootingCalculationsCommand()
+        );
+    }
+
+    public static Command getFeedNoteForShootingCommand() {
+        return runWhen(IntakeCommands.getSetTargetStateCommand(IntakeConstants.IntakeState.FEED_SHOOTING), () -> RobotContainer.SHOOTER.atTargetVelocity() && RobotContainer.PITCHER.atTargetPitch() && RobotContainer.SWERVE.atAngle(SHOOTING_CALCULATIONS.getTargetShootingState().targetRobotAngle()));
+    }
+
+    public static Command getCloseSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                getPrepareCloseSpeakerShotCommand(),
+                getFeedNoteForCloseSpeakerShotCommand()
+        );
+    }
+
+    public static Command getPrepareCloseSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                PitcherCommands.getSetTargetPitchCommand(PitcherConstants.CLOSE_SHOT_PITCH),
+                ShooterCommands.getSetTargetVelocityCommand(ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND * ShooterConstants.RIGHT_MOTOR_TO_LEFT_MOTOR_RATIO)
+        );
+    }
+
+    public static Command getFeedNoteForCloseSpeakerShotCommand() {
+        return runWhen(IntakeCommands.getSetTargetStateCommand(IntakeConstants.IntakeState.FEED_SHOOTING), () -> RobotContainer.SHOOTER.atTargetVelocity() && RobotContainer.PITCHER.atTargetPitch());
+    }
+
+    public static Command getAutonomousScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getAutonomousPrepareForAmp(),
+                runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                getPathfindToAmpCommand().andThen(duplicate(CommandConstants.FACE_AMP_COMMAND))
+        );
+    }
+
+    public static Command getScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getPrepareForAmpCommand(),
+                runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                duplicate(CommandConstants.FACE_AMP_COMMAND)
+        );
+    }
+
+    public static Command getAutonomousPrepareForAmp() {
```

Comment:
Explain the difference between the autnomous preparation and normal one, and mention that this is a preparation only for a real time amp (rather than the actaul 15 seconds auton portion)